src/shadowbox/server/mocks/mocks.ts

```diff
@@ -56,15 +56,8 @@ export class FakePrometheusClient extends PrometheusClient {
 
   async query(query: string): Promise<QueryResultData> {
     const accessKeyIdMatch = query.match(/access_key="(.*?)"/);
-    let accessKeyIds: string[];
-    if (accessKeyIdMatch && accessKeyIdMatch[1]) {
-      // Return query results only for the specified access key.
-      accessKeyIds = [accessKeyIdMatch[1]];
-    } else {
-      accessKeyIds = Object.keys(this.bytesTransferredById);
-    }
     const queryResultData = {result: []} as QueryResultData;
-    for (const accessKeyId of accessKeyIds) {
+    for (const accessKeyId of Object.keys(this.bytesTransferredById)) {
```

Comment:
Can you use `for(const accessKeyId in this.bytesTransferredById)` instead?

Reply:
Technically yes, but the compiler complains with: `for (... in ...) statements must be filtered with an if statement`. `Object.keys` avoids the check.

Reply:
Yuck.  OK.